### ISSUE TITLE
Update parso to v0.2

### ISSUE THREAD

**Issue by lqmanh** [palantir/python-language-server#337]
When trying to run `pyls --help`, I encountered this error:
```
pkg_resources.DistributionNotFound: The 'parso==0.1.1' distribution was not found and is required by jedi
```
However, I already have parso 0.2.0 installed in my system.

**Comment by gatesn** [comment]
@lqmanh as per this PR https://github.com/davidhalter/jedi/commit/bcd05f560e39b94946f1d337374fb60001e35a2a, parso should be compatible with 0.2 as of `python-language-server==0.17.2`

**Comment by gatesn** [comment]
Sorry, I read this wrong. We need to fix #334 for pyls to be compatible with parso >= 0.2
